src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,65 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+
+/**
+ * Handles fetching and saving user data.
+ */
+@WebServlet("/about")
+public class AboutMeServlet extends HttpServlet {
```

Comment:
It looks like the servlet operates primarily on HTML pages. 

To allow for programmatic usage (e.g. with an Android or iOS app, with JavaScript, or with some other client-code), it is a good idea to expose the functionality using a REST-ful API (i.e. operating on JSON rather than HTML) and to render the content on the client (e.g. using JavaScript) or to have a second servlet that does HTML rendering on the server in addition to the REST API (e.g. for cases where JavaScript is disabled).